client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -1,21 +1,27 @@
-import { FunctionComponent } from 'react';
-import type { TranslateResult } from 'i18n-calypso';
+/* eslint-disable wpcalypso/jsx-classname-namespace */
 
-export type EmailProviderFeatures = {
-	header: string;
-	tools: TranslateResult;
-	storage: TranslateResult;
-	importing: TranslateResult;
-	support: TranslateResult;
-	selectCallback: () => void;
-};
+import { FunctionComponent } from 'react';
```

Comment:
Should this not be `import type`?

Reply:
Good catch. Actually, it looks we have many imports in the `client/my-sites/email` folder that need to be fixed. I will take care of that in another pull request though (and address your [other comment](https://github.com/Automattic/wp-calypso/pull/59714/files#r778290020) in the process).

Reply:
This will be addressed in https://github.com/Automattic/wp-calypso/pull/59775.